sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -841,17 +828,21 @@ export class CertificateClient {
     issuerName: string,
     options: UpdateIssuerOptions = {}
   ): Promise<CertificateIssuer> {
+    const { span, updatedOptions } = createSpan("updateIssuer", options);
+
+    // TODO: @sadasant - this covers for a bug where UpdateIssuerOptions doesn't
```

Comment:
Future TODO: 

Bug where `RequestOptionsBase` was allowing for any field to work but the field is not actually available in the public options for the method.

CC: @sadasant , @maorleger